src/components/landing/rework/landing-subjects-new.tsx

```diff
@@ -0,0 +1,312 @@
+import { faArrowCircleRight } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { lighten } from 'polished'
+import styled, { keyframes } from 'styled-components'
+
+import AbcSVG from '@/assets-webkit/img/landing/subjects-abc.svg'
+import BiologySVG from '@/assets-webkit/img/landing/subjects-biology.svg'
+import BlankSVG from '@/assets-webkit/img/landing/subjects-blank.svg'
+import ChemistrySVG from '@/assets-webkit/img/landing/subjects-chemistry.svg'
+import InformaticsSVG from '@/assets-webkit/img/landing/subjects-informatics.svg'
+import MathSVG from '@/assets-webkit/img/landing/subjects-math.svg'
+import NewSVG from '@/assets-webkit/img/landing/subjects-new.svg'
+import SustainabilitySVG from '@/assets-webkit/img/landing/subjects-sustainability.svg'
+import { Link } from '@/components/content/link'
+import {
+  LandingSubjectIcon,
+  LandingSubjectLink,
+  LandingSubjectsData,
+} from '@/data-types'
+import { makeResponsivePadding, makeTransparentButton } from '@/helper/css'
+
+interface LandingSubjectsProps {
+  data: LandingSubjectsData
+}
+
+export function LandingSubjectsNew({ data }: LandingSubjectsProps) {
+  return (
+    <>
+      <SubjectsWrapper>{data.subjects.map(renderSubject)}</SubjectsWrapper>
+
+      {data.additionalLinks.length > 0 && (
```

Comment:
@elbotho Are there any additional links in the new landing page design?

Reply:
Not at this point, but I opted to keep the code since the design is not fixed atm.
If you want you can mark it with a comment to remove it when we remove the "old" landing page completely.

Reply:
Or remove it now.